contracts/implementation/LeveragedPool.sol

```diff
@@ -207,10 +203,12 @@ contract LeveragedPool is ILeveragedPool, Initializable {
             (uint256 newLongBalance, uint256 newShortBalance, uint256 totalFeeAmount) = PoolSwapLibrary
                 .calculatePriceChange(priceChangeData);
 
-            emit PoolRebalance(
-                int256(newShortBalance) - int256(_shortBalance),
-                int256(newLongBalance) - int256(_longBalance)
-            );
+            unchecked {
+                emit PoolRebalance(
+                    int256(newShortBalance) - int256(_shortBalance),
+                    int256(newLongBalance) - int256(_longBalance)
+                );
+            }
```

Comment:
Very wary of `unchecked` blocks; what's the reasoning for why this block is safe to turn the checker off here?

Reply:
It can not underflow due to the types being `int256`, allowing for negative numbers.

Reply:
Can you elaborate? `_shortBalance` can be greater than `newShortBalance`, and thus you would have a negative number when subtracting the latter from the former.

Reply:
Understood in DM's